types/react/jsx-runtime.d.ts

```diff
@@ -0,0 +1,3 @@
+import { createElement } from './';
+
+export default createElement;
```

Comment:
That's almost certainly not correct but at least it gets some type-checking. We'll have to figure out later what the correct approach would be. Personally, I would've preferred we introduce it in a separate PR (since older majors also need it) but otherwise the PR is blocked.

Reply:
If you plan to work on those entries as part of other PR I would rather recommend removing this entirely - at least the `createElement` export because as you have said: it's not correct and just won't match the expectations.

I'm not 100% sure about the TS resolution algorithm because they are somewhat magic to me but what it might do right now is just pulling the root entry (`index`) and thus bringing those types into the project (the global ones) that might make its global JSX namespace work. OTOH those types should probably be already pulled by other means so not sure if this is at all needed here (if you plan to work on this in a separate PR).